src/IsaacApiTypes.tsx

```diff
@@ -511,7 +511,7 @@ export type Stage = "year_7_and_8" | "year_9" | "gcse" | "a_level" | "further_a"
 
 export type ExamBoard = "aqa" | "cie" | "edexcel" | "eduqas" | "ocr" | "wjec" | "all";
 
-export type Difficulty = "practice_1" | "practice_2" | "practice_3" | "challenge_1" | "challenge_2" | "challenge_3";
```

Comment:
Are these difficulties specifically associated with physics for some reason or has cs just not made use of them so far?

Reply:
Dug into this some more and had a play about. I've reverted it to how it was before (and also for Constants.ts) to preserve for future functionality, and it's easy to get Practice 3 and Challenge 3 to correctly display questions in the Question Finder for CS, however the icons are not correct so I'm going to create a separate ticket for this one.  It looks like there's only 4 "practice_3" questions for CS and 13 "challenge_3" so it's impacting very little content compared to the other difficulty levels (hundreds of questions for each)